snowplow/iglu-client-embedded/schemas/com.metabase/instance_stats/jsonschema/1-0-0

```diff
@@ -35,6 +35,32 @@
         ]
       }
     },
+    "metrics": {
+      "description": "Key-value pairs of metrics",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "description": "A single metric attribute",
+        "properties": {
+          "key": {
+            "description": "The key for this attribute",
+            "type": "string",
+            "maxLength": 255
+          },
+          "value": {
+            "description": "The value of this attribute",
+            "type": ["string", "boolean", "integer", "null"],
+            "maxLength": 255,
+            "minimum": 0,
+            "maximum": 2147483647
+          }
+        },
+        "required": [
+          "key",
+          "value"
+        ]
+      }
+    },
```

Comment:
**nitpick (blocking):** We need to make this a new minor schema version `1-0-1`.

Context: In SnowcatCloud the schemas cannot be edited when they are deployed already. This means if we change it here and start sending new properties (i.e. `metrics`), the event will be rejected. So if we do updates after deployment, they need to be new schema versions that we then deploy again in SnowcatCloud 🙂

Reply:
Thank you for pointing this out! 

I will revert this file, and add the `"metrics"` key to an identical json file in `metabase/snowplow/iglu-client-embedded/schemas/com.metabase/instance_stats/jsonschema/1-0-1`. 

I think I also need to bump the value for `::instance_stats` to `"1-0-1"` [here](https://github.com/metabase/metabase/blob/add-metrics-to-snowplow-ping/src/metabase/analytics/snowplow.clj#L35-L41). But I'm not sure: Do you know when a new version of a schema gets added to SnowcatCloud? Are there some docs for this? I'll look around when I get a chance.

Reply:
> I think I also need to bump the value for ::instance_stats to "1-0-1"
Yes I think that, too (but @qnkhuat) might have better knowledge and can confirm.
[Here](https://www.notion.so/metabase/Snowplow-integration-5da1f874beda4153b4fccfa6c1e77caa?pvs=4#723b7505593c4f7fa1e29416a2377ffa) is a bit of docs around the process but I think we need to still improve it a bit (feel free to adjust it based on your learnings). 

I adjusted a small [section](https://www.notion.so/metabase/Snowplow-integration-5da1f874beda4153b4fccfa6c1e77caa?pvs=4#50e90bb17b27409da20a68a48251bf42) to give some indication on when to talk to SnowcatCloud. TL;DR: When you have the PR up and we internally agreed on the schema format, we should already sanity check with them before merging. Sometimes Joao has some last minute concerns. I was pushing for us (data team) having more ownership and being able to update the schemas. That worked for a week and now we don't have the permissions anymore since their system is not very error proof sometimes.

Reply:
fixed. Thanks for the links